src/components/SessionsView.jsx

```diff
@@ -0,0 +1,62 @@
+import tableStyles from '../styles/table'
+import viewStyles from '../styles/view'
+
+import React from 'react'
+import classNames from 'classnames'
+import parser from 'user-agent-parser'
+
+import Loading from './Loading'
+
+import { translate } from 'cozy-ui/react/helpers/i18n'
+
+const SessionsView = ({ t, f, isFetching, sessions, deleteOtherSessions }) => {
+  return (
+    <div role='contentinfo'>
+      <h2 className={viewStyles['set-view-title']}>
+        {t('SessionsView.title')}
+      </h2>
+      <p className={viewStyles['set-view-title']}><button className={classNames('coz-btn', 'coz-btn--danger')} onClick={() => deleteOtherSessions()}>{t('SessionsView.delete')}</button></p>
+      { isFetching && <Loading /> }
+      { !isFetching && sessions && (
+        <div className={classNames(
+          tableStyles['coz-table']
+        )}>
+          <div className={classNames(tableStyles['coz-table-head'], tableStyles['coz-table-row'])}>
+            <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-date'])}>Date</div>
+            <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-os'])}>OS</div>
+            <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-browser'])}>Browser</div>
+            <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-ip'])}>IP</div>
+          </div>
+          <div className={classNames(tableStyles['coz-table-body'])}>
+            {
+              sessions
+              .map(session => (
+                <div className={tableStyles['coz-table-row']}>
+                  <div className={classNames(tableStyles['coz-table-cell'], tableStyles['set-table-date'])}>
+                    {
+                      f(
+                        new Date(session.created_at),
+                        t('SessionsView.sync_date_format')
+                      )
+                    }
+                  </div>
+                  <div className={classNames(tableStyles['coz-table-cell'], tableStyles['set-table-os'])}>
+                    {session.os}
+                  </div>
+                  <div className={classNames(tableStyles['coz-table-cell'], tableStyles['set-table-browser'])}>
+                    {parser(session.user_agent).browser.name} {parser(session.user_agent).browser.major}
```

Comment:
Maybe parsing the `session.user_agent` only once before the `render` function could be better, what do you think?

Reply:
session is defined in the map function, I can't access it in the render. 
But I can make a sub component <SessionViewRow session={session} /> :+1: